app/models/support_interface/vendor_api_monitor.rb

```diff
@@ -26,21 +26,21 @@ def no_sync_in_24h
       connected
         .select('last_syncs.last_sync as last_sync, vendor_id')
         .joins("LEFT JOIN (#{VendorAPIRequest.successful.syncs.select('provider_id, MAX(vendor_api_requests.created_at) as last_sync').group('provider_id').to_sql}) last_syncs on last_syncs.provider_id = providers.id")
-        .where("last_sync < now() - interval '24 hours' OR last_sync IS NULL").order('last_sync DESC')
+        .where("last_sync < ('#{Time.zone.now.utc.iso8601(6)}'::timestamp - interval '24 hours') OR last_sync IS NULL").order('last_sync DESC')
```

Comment:
should we extract this to `postgres_current_time` or similar?

Reply:
Done (pending test fixes)